src/melee/ft/chara/ftMario/ftMario_SpecialHi.c

```diff
@@ -22,7 +22,7 @@ void ftMario_SpecialAirHi_StartAction(HSD_GObj* gobj)
 {
     Fighter* fp;
     ftMarioAttributes* sa;
-    u8 unused[4];
+    char unused[4];
```

Comment:
The rationale for `u8` to `char` here is a little hard to explain but ask me on Discord if needed.